src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -233,876 +233,926 @@ void GetPurchasedAir(EnergyPlusData &state)
-                            Constant::eResource::DistrictHeatingWater,
-                            OutputProcessor::Group::HVAC,
-                            OutputProcessor::EndUseCat::Heating);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Supply Air Sensible Cooling Energy",
                             Constant::Units::J,
-                            PurchAir(PurchAirNum).SenCoolEnergy,
+                            PurchAir.SenCoolEnergy,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Supply Air Latent Cooling Energy",
                             Constant::Units::J,
-                            PurchAir(PurchAirNum).LatCoolEnergy,
+                            PurchAir.LatCoolEnergy,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name);
+                            PurchAir.Name);
         SetupOutputVariable(state,
                             "Zone Ideal Loads Supply Air Total Cooling Energy",
                             Constant::Units::J,
-                            PurchAir(PurchAirNum).TotCoolEnergy,
+                            PurchAir.TotCoolEnergy,
                             OutputProcessor::TimeStepType::System,
                             OutputProcessor::StoreType::Sum,
-                            PurchAir(PurchAirNum).Name,
-                            Constant::eResource::DistrictCooling,
+                            PurchAir.Name);
+        SetupOutputVariable(state,
+                            "Zone Ideal Loads Supply Air Total Heating Fuel Energy",
+                            Constant::Units::J,
+                            PurchAir.TotHeatFuelEnergy,
+                            OutputProcessor::TimeStepType::System,
+                            OutputProcessor::StoreType::Sum,
+                            PurchAir.Name,
+                            Constant::eFuel2eResource[(int)PurchAir.heatingFuelType],
+                            OutputProcessor::Group::HVAC,
+                            OutputProcessor::EndUseCat::Heating);
+        SetupOutputVariable(state,
+                            "Zone Ideal Loads Supply Air Total Cooling Fuel Energy",
+                            Constant::Units::J,
+                            PurchAir.TotCoolFuelEnergy,
+                            OutputProcessor::TimeStepType::System,
+                            OutputProcessor::StoreType::Sum,
+                            PurchAir.Name,
+                            Constant::eFuel2eResource[(int)PurchAir.coolingFuelType],
```

Comment:
The `eResource` types are set in the energy report variables, instead of the load report variables.